logger/logger.go

```diff
@@ -88,15 +89,23 @@ func Debugf(format string, v ...interface{}) {
 func Log(l *log.Logger, color string, v ...interface{}) {
 	msg := fmt.Sprint(v...)
 	if Color {
-		msg = color + msg + ColorNC
+		msg = colorizeMessage(color, msg)
 	}
 	l.Output(calldepth, msg)
 }
 
 func Logf(l *log.Logger, color, format string, v ...interface{}) {
 	msg := fmt.Sprintf(format, v...)
 	if Color {
-		msg = color + msg + ColorNC
+		msg = colorizeMessage(color, msg)
 	}
 	l.Output(calldepth, msg)
 }
+
+func colorizeMessage(color, s string) string {
+	whitespace := regexp.MustCompile(`\s*$`)
```

Comment:
I wouldn't use `regexp.MustCompile` as it will panic, if for example, the regex is changed and is wrong. 

Must better to use `whitespace err := regexp.Compile(`\s*$`)` and handle the error

Reply:
No, I specifically want to ensure that the regex is valid so panicing should a developer break it needs to be flagged up powerfully.  If the regex was being supplied by a user I would consider using regex.Compile though. :)

Reply:
I see, in that case then. it's all good

Reply:
I see, in that case then. it's all good